packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx

```diff
@@ -18,7 +18,16 @@ class EmptyDocumentHolder extends Component {
   }
 
   async onChange(file) {
-    const { documentId, client, linkDocumentSuccess, t } = this.props
+    const {
+      documentId,
+      client,
+      linkDocumentSuccess,
+      t,
+      index,
+      fetchDocumentLoading,
+      fetchDocumentError
+    } = this.props
+    fetchDocumentLoading({ idDoctemplate: documentId, index })
```

Comment:
The `fetch` prefix in this context seem wrong, since there's no real fetching happening here.